src/test/scala/mesosphere/marathon/api/v2/TasksResourceTest.scala

```diff
@@ -91,6 +91,51 @@ class TasksResourceTest extends MarathonSpec with GivenWhenThen with Matchers wi
     noMoreInteractions(taskKiller)
   }
 
+  test("killTasks with scale and wipe fails") {
+    Given("a request")
+    val app1 = "/my/app-1".toRootPath
+    val taskId1 = Task.Id.forApp(app1).idString
+    val body = s"""{"ids": ["$taskId1"]}"""
+    val bodyBytes = body.toCharArray.map(_.toByte)
+
+    When("we ask to scale AND wipe")
+    val exception = intercept[BadRequestException] {
+      taskResource.killTasks(scale = true, force = false, wipe = true, body = bodyBytes, auth.request)
+    }
+
+    Then("an exception should occur")
+    exception.getMessage shouldEqual "You cannot use scale and wipe at the same time."
+  }
+
+  // FIXME (3221): breaks – why?
+  ignore("killTasks with wipe delegates to taskKiller with wipe value") {
```

Comment:
Did you test this on a real Marathon instance?
Is that "just" a problem of the test setup?